### PR TITLE
Update 5-semantic-layer-marts.md

### PR DESCRIPTION
**Pull request by mirnawong1**
adding semantic models configuration guidance and naming guidance per slack thread: https://dbt-labs.slack.com/archives/C03KHQRQUBX/p1716313470074679

### CODE REVIEW

File: website/docs/best-practices/how-we-structure/5-semantic-layer-marts.md [hunk 1]
```diff
@@ -36,6 +36,17 @@ models
     └── stg_supplies.yml
 ```
 
+## Semantic Layer: Where and why?
+
+- 📂 Directory structure: Add your semantic models in `models/semantic_models` with directories corresponding to the models/marts files. This organization makes it easier to search and find what you can join. It also supports better maintenance and reduces repeated code.
+
+## Naming conventions
```
Comment (mirnawong1) [inline website/docs/best-practices/how-we-structure/5-semantic-layer-marts.md]:
```suggestion
## Naming convention
```

File: website/docs/best-practices/how-we-structure/5-semantic-layer-marts.md [hunk 2]
```diff
@@ -36,6 +36,17 @@ models
     └── stg_supplies.yml
 ```
 
+## Semantic Layer: Where and why?
+
+- 📂 Directory structure: Add your semantic models in `models/semantic_models` with directories corresponding to the models/marts files. This organization makes it easier to search and find what you can join. It also supports better maintenance and reduces repeated code.
+
```
Comment (mirnawong1) [inline website/docs/best-practices/how-we-structure/5-semantic-layer-marts.md:65]: ADD EXAMPLE

**Comment by mirnawong1** [comment]
need to link to this page from other docs

File: website/docs/best-practices/how-we-structure/5-semantic-layer-marts.md [hunk 3]
```diff
@@ -36,6 +36,40 @@ models
     └── stg_supplies.yml
 ```
 
+## Semantic Layer: Where and why?
+
+- 📂 **Directory structure**: Add your semantic models in the `models/semantic_models` with directories corresponding to the models/marts files. This type of organization makes it easier to search and find what you can join. It also supports better maintenance and reduces repeated code.
```
Comment (matthewshaver) [inline website/docs/best-practices/how-we-structure/5-semantic-layer-marts.md]:
```suggestion
- 📂 **Directory structure**: Add your semantic models to `models/semantic_models` with directories corresponding to the models/marts files. This type of organization makes it easier to search and find what you can join. It also supports better maintenance and reduces repeated code.
```

File: website/docs/best-practices/how-we-structure/5-semantic-layer-marts.md [hunk 4]
```diff
@@ -36,6 +36,40 @@ models
...
+            type: time
+            type_params:
+              time_granularity: day
+    ```
+    </File>
+
+## Naming convention
+
+- 🏷️ **Semantic model names**: Use the `sem_` prefix for semantic model names, such as `sem_cloud_user_account_activity`. This follows the same pattern as other naming conventions like `fct_` for fact tables and `dim_` for dimension tables.
+- 🧩 **Entity names**: Entity names don't use prefixes within the semantic model. This keeps the names clear and focused on their specific purpose without unnecessary prefixes.
```
Comment (matthewshaver) [inline website/docs/best-practices/how-we-structure/5-semantic-layer-marts.md]:
```suggestion
- 🧩 **Entity names**: Don't use prefixes in Entity names within the semantic model. This keeps the names clear and focused on their specific purpose without unnecessary prefixes.
```

**Review by matthewshaver — APPROVED** [review]

**Pull request merged**